docs/content/en/partials/_index-deployment-checks-left.md

```diff
@@ -1,7 +1,5 @@
 
-## Deployment Check Orchestration
-
-### Orchestrating deployment checks
+## Orchestrate Deployment Checks
 
 To reduce complexity of custom checks use Keptn to:
```

Comment:
This prose and the rest of the content in this section is absolutely non-sensical!  I favor omitting this whole block from the page, actually, and just going directly to the information in https://main.lifecycle.keptn.sh/docs/intro/ -- we can link to the appropriate sub-page for each use case, and those link to the appropriate Getting Started exercise and Guide chapter.

Reply:
any big changes to the landing page will be done later after we did the first rounds of quick fixes and the second round of user interviews to test them out.